StratisCore.UI/src/app/shared/services/current-account.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, Observable } from "rxjs";
+
+/*
+ * Used to maintain the state of the currently selected address in CirrusCore.
+ */
+@Injectable({
+  providedIn: 'root'
+})
+export class CurrentAccountService {
+
+  private _currentAddress: BehaviorSubject<string> = new BehaviorSubject<string>(null);
+
+  constructor() {
+  }
+
+  public get currentAddress(): Observable<string> {
+    return this._currentAddress.asObservable();
+  }
+
+  public getAddress(): string {
```

Comment:
can we change to `get address` and `set address`

Reply:
Don't understand this. We have a get and set address on here. the `get currentAddress()` method returns the `BehaviourSubject` as an Observable

Reply:
I mean changing it from a method to use a property so changing `getAddress` to `get address` and `setAddress` to `set address`, not a big one? Just a nice to have